common/management/commands/prune_unused_wagtail_tags.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from taggit.models import Tag
+
+
+class Command(BaseCommand):
+    help = 'Delete wagtail tags that are not applied to any items'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--commit',
+            action='store_true',
+            help='Commit changes to the database',
+        )
```

Comment:
test comment, please ignore

```suggestion
    def add_arguments(self, parser):
        parser.add_argument(
            '--commit',
            action='store_true',
            help='Commit changes to the database',
        ) 
```

Reply:
(Sorry, was just testing the behavior of suggestions. This is not a real suggestion.)